src/rpc/blockchain.cpp

```diff
@@ -953,7 +950,7 @@ static UniValue pruneblockchain(const JSONRPCRequest& request)
     return uint64_t(height);
 }
 
-static UniValue gettxoutsetinfo(const JSONRPCRequest& request)
+static UniValue gettxoutsetinfo(const JSONRPCRequest& request) EXCLUSIVE_LOCKS_REQUIRED(cs_main)
```

Comment:
This is incorrect and ignored by clang anyway

Reply:
@MarcoFalke In what way? Doesn’t reading the variable `pcoinsdbview` require holding the mutex `cs_main`?

Reply:
All the rpc calls are called through function pointers, which breaks clangs static analyser. Clearly there is no way to acquire this lock in the caller of `gettxoutsetinfo`. If the lock is missing, it should be added...